src/base.iced

```diff
@@ -88,6 +90,24 @@ class Verifier
 
   #---------------
 
+  _get_now : () -> @now or unix_time()
```

Comment:
It looks like `now` isn't allowed to be `0` this way. Even though Go uses zero as the magical "not actually a time" value sometimes, maybe it would be cleaner to only treat `undefined`/`null` that way here (`if @now? then @now else unix_time()`), so that all numbers behave the same?